lib/assets/javascripts/cartodb/table/mapview.js

```diff
@@ -1295,6 +1318,14 @@ cdb.admin.MapTab = cdb.core.View.extend({
     this.mapView.$el.append(this.stackedLegend.render().$el);
     this.addView(this.stackedLegend);
     if (this.overlays) this.overlays.setLegend(this.stackedLegend);
+
+    if (this.legends && this.legends.length) {
+      // restore legend CSS if placed in a custom spot
```

Comment:
Aren't we going to need this in cartodb.js as well so that the public maps properly place the legends at their new positions?